<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>937</width>
    <height>569</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="frame_superior">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>35</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>50</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">
QFrame{
border-top-left-radius: 20px;
border-bottom-left-radius: 20px;

background-color: rgb(170, 0, 255);

}
</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_8">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>2</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>2</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item alignment="Qt::AlignLeft">
        <widget class="QPushButton" name="bt_menu">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
background-color: #aa00ff;
font: 87 12pt &quot;Arial Black&quot;;
border-radius:0px;
}

QPushButton:hover{
background-color: white;
font: 87 12pt &quot;Arial Black&quot;;

}
</string>
         </property>
         <property name="text">
          <string>    MENU </string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>../../Pruebas_para_Proyecto_Practicas/Menu_Lateral_desplegable_1/PyQt5-main/PyQt5-main/Menu Lateral desplegable/barra-de-menus.png</normaloff>../../Pruebas_para_Proyecto_Practicas/Menu_Lateral_desplegable_1/PyQt5-main/PyQt5-main/Menu Lateral desplegable/barra-de-menus.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="autoDefault">
          <bool>false</bool>
         </property>
         <property name="default">
          <bool>false</bool>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>265</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item alignment="Qt::AlignRight">
        <widget class="QPushButton" name="bt_minimizar">
         <property name="minimumSize">
          <size>
           <width>35</width>
           <height>35</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
border:0px;
}

QPushButton:hover{
border:5px solid #aa00ff;
background-color:#ffff00;

}
</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>../icons/minimizar.png</normaloff>../icons/minimizar.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item alignment="Qt::AlignRight">
        <widget class="QPushButton" name="bt_restaurar">
         <property name="maximumSize">
          <size>
           <width>35</width>
           <height>35</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
border:0px;

}

QPushButton:hover{
border:5px solid #aa00ff;
background-color:#55ff00;

}
</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>../icons/restaurar.png</normaloff>../icons/restaurar.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
        </widget>
       </item>
       <item alignment="Qt::AlignRight">
        <widget class="QPushButton" name="bt_maximizar">
         <property name="maximumSize">
          <size>
           <width>35</width>
           <height>35</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
border:0px;

}

QPushButton:hover{
border:5px solid #aa00ff;
background-color:#55ff00;

}
</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>../icons/maximizar.png</normaloff>../icons/maximizar.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
        </widget>
       </item>
       <item alignment="Qt::AlignRight">
        <widget class="QPushButton" name="bt_cerrar">
         <property name="maximumSize">
          <size>
           <width>35</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
border:0px;

}

QPushButton:hover{
border:5px solid #aa00ff;
background-color:red;

}
</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>../icons/cerrar.png</normaloff>../icons/cerrar.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frame_inferior">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item alignment="Qt::AlignLeft">
        <widget class="QFrame" name="frame_lateral">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>0</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="styleSheet">
          <string notr="true">QFrame {
    background-color: #aa55ff;
}

QPushButton {
    background-color: #aa55ff;
    border: 2px solid #fff;
    border-top-left-radius: 20px;
    border-bottom-left-radius: 20px;
    padding: 10px;
    color: white;
    font: 75 12pt &quot;Arial Narrow&quot;;
    text-align: left;
}

QPushButton:hover {
    background-color: #ffffff;
    border: 2px solid #aa55ff;
    color: #aa55ff;
    text-align: left;
}
</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Plain</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <property name="spacing">
           <number>20</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>9</number>
          </property>
          <item>
           <widget class="QPushButton" name="bt_inicio">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>40</height>
             </size>
            </property>
            <property name="contextMenuPolicy">
             <enum>Qt::DefaultContextMenu</enum>
            </property>
            <property name="layoutDirection">
             <enum>Qt::LeftToRight</enum>
            </property>
            <property name="text">
             <string>       INICIO             </string>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>../icons/inteligencia-artificial.png</normaloff>../icons/inteligencia-artificial.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>32</width>
              <height>32</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="bt_uno">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>40</height>
             </size>
            </property>
            <property name="toolTipDuration">
             <number>0</number>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string>USUARIOS</string>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>../icons/base-de-datos.png</normaloff>../icons/base-de-datos.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>32</width>
              <height>32</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="bt_dos">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>40</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string>PELICULAS</string>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>../icons/chip.png</normaloff>../icons/chip.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>32</width>
              <height>32</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="bt_tres">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>40</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string>FUNCIONES</string>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>../../Pruebas_para_Proyecto_Practicas/Menu_Lateral_desplegable_1/PyQt5-main/PyQt5-main/Menu Lateral desplegable/moto.png</normaloff>../../Pruebas_para_Proyecto_Practicas/Menu_Lateral_desplegable_1/PyQt5-main/PyQt5-main/Menu Lateral desplegable/moto.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>32</width>
              <height>32</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="bt_cuatro">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>40</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string>HISTORIAL</string>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>../../Pruebas_para_Proyecto_Practicas/Menu_Lateral_desplegable_1/PyQt5-main/PyQt5-main/Menu Lateral desplegable/encriptado.png</normaloff>../../Pruebas_para_Proyecto_Practicas/Menu_Lateral_desplegable_1/PyQt5-main/PyQt5-main/Menu Lateral desplegable/encriptado.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>32</width>
              <height>32</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="bt_cinco">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>40</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string>ESTADISTICA</string>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>../../Pruebas_para_Proyecto_Practicas/Menu_Lateral_desplegable_1/PyQt5-main/PyQt5-main/Menu Lateral desplegable/videojuego.png</normaloff>../../Pruebas_para_Proyecto_Practicas/Menu_Lateral_desplegable_1/PyQt5-main/PyQt5-main/Menu Lateral desplegable/videojuego.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>32</width>
              <height>32</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="label_2">
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string>INTENTO 1</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QStackedWidget" name="stackedWidget">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Plain</enum>
         </property>
         <property name="lineWidth">
          <number>0</number>
         </property>
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="page">
          <layout class="QVBoxLayout" name="verticalLayout_9">
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string/>
             </property>
             <property name="pixmap">
              <pixmap>../icons/download.jpg</pixmap>
             </property>
             <property name="scaledContents">
              <bool>false</bool>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
             <property name="indent">
              <number>0</number>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="page_uno">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 255, 255);</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QWidget" name="widget_2" native="true">
             <layout class="QVBoxLayout" name="verticalLayout_5">
              <item>
               <widget class="QLabel" name="label_3">
                <property name="text">
                 <string>Usuarios</string>
                </property>
               </widget>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_3">
                <item>
                 <widget class="QLineEdit" name="lineEdit"/>
                </item>
                <item>
                 <widget class="QPushButton" name="pushButton_5">
                  <property name="text">
                   <string>PushButton</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="pushButton_8">
                  <property name="text">
                   <string>Modificar</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <widget class="QTableWidget" name="tableWidget">
                <row>
                 <property name="text">
                  <string>1</string>
                 </property>
                </row>
                <row>
                 <property name="text">
                  <string>2</string>
                 </property>
                </row>
                <row>
                 <property name="text">
                  <string>3</string>
                 </property>
                </row>
                <row>
                 <property name="text">
                  <string>4</string>
                 </property>
                </row>
                <row>
                 <property name="text">
                  <string>5</string>
                 </property>
                </row>
                <row>
                 <property name="text">
                  <string>6</string>
                 </property>
                </row>
                <row>
                 <property name="text">
                  <string>7</string>
                 </property>
                </row>
                <column>
                 <property name="text">
                  <string>IdUsuario</string>
                 </property>
                </column>
                <column>
                 <property name="text">
                  <string>Contraseña</string>
                 </property>
                </column>
                <column>
                 <property name="text">
                  <string>Grupo</string>
                 </property>
                </column>
                <column>
                 <property name="text">
                  <string>Fecha_Creacion</string>
                 </property>
                </column>
                <column>
                 <property name="text">
                  <string>ETC</string>
                 </property>
                </column>
                <column>
                 <property name="text">
                  <string>etc</string>
                 </property>
                </column>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="page_dos">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 255, 255);</string>
          </property>
          <widget class="QPushButton" name="pushButton_4">
           <property name="geometry">
            <rect>
             <x>9</x>
             <y>246</y>
             <width>75</width>
             <height>76</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(255, 0, 127);
font: 87 36pt &quot;Arial Black&quot;;</string>
           </property>
           <property name="text">
            <string>2</string>
           </property>
          </widget>
         </widget>
         <widget class="QWidget" name="page_tres">
          <widget class="QPushButton" name="pushButton_3">
           <property name="geometry">
            <rect>
             <x>9</x>
             <y>246</y>
             <width>75</width>
             <height>76</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(255, 255, 0);
font: 87 36pt &quot;Arial Black&quot;;</string>
           </property>
           <property name="text">
            <string>3</string>
           </property>
          </widget>
         </widget>
         <widget class="QWidget" name="page_cuatro">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 255, 255);</string>
          </property>
          <widget class="QPushButton" name="pushButton_2">
           <property name="geometry">
            <rect>
             <x>9</x>
             <y>246</y>
             <width>75</width>
             <height>76</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(85, 255, 0);
font: 87 36pt &quot;Arial Black&quot;;</string>
           </property>
           <property name="text">
            <string>4</string>
           </property>
          </widget>
         </widget>
         <widget class="QWidget" name="page_cinco">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 255, 255);</string>
          </property>
          <widget class="QPushButton" name="pushButton">
           <property name="geometry">
            <rect>
             <x>9</x>
             <y>228</y>
             <width>75</width>
             <height>76</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(0, 170, 255);
font: 87 36pt &quot;Arial Black&quot;;</string>
           </property>
           <property name="text">
            <string>5</string>
           </property>
          </widget>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_contenedor">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <property name="lineWidth">
          <number>1</number>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
